tensorflow/java/src/main/java/org/tensorflow/Graph.java

```diff
@@ -77,6 +80,22 @@ public Operation operation(String name) {
   }
 
   /**
+   * Returns all operations that exist in the Graph.
```

Comment:
Let's clarify that there is no significance to the order of the returned operations. Perhaps something like:

```java
* The {@link Operation} objects are returned in an underfined order.
```